lib/puppet/defaults.rb

```diff
@@ -196,7 +196,7 @@ def self.initialize_default_settings!(settings)
       end
     },
     :disable_i18n => {
-      :default => false,
+      :default => true,
```

Comment:
I generally try to keep commits into separate logic units, so one commit to make the locale pluginsync observe the setting and another to change the setting's default value.